fabric-rendering-v0/src/main/java/net/fabricmc/fabric/api/client/event/HudRenderCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.client.event;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	/**
+	 * Called after rendering the status effects overlay when rendering the hud.
+	 */
+	Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class, (listeners) ->
+			(delta) -> {
+		for (HudRenderCallback event : listeners) {
+			event.render(delta);
+		}
+	});
+
+	/**
+	 * Called when the hud is rendered, which is displayed in game, in a world.
+	 * See {@link net.minecraft.util.math.MathHelper#lerp(float, float, float)} for interpolating.
+	 */
+	void render(float delta);
```

Comment:
onHudRender IMO, please also document when it is being called in ralation to vanilla's own hud rendering and rename the parameter to tickDelta to make it clear what it means. The See part of the description should use the \@see tag and it's missing a blank line after the first sentence.